about/contributing_datasets.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: default
+title: Contributing datasets
+nav_order: 1
+parent: About Data Commons
+---
+
+# Life of a Dataset
+
+This tutorial walks through the process of structuring and inserting data into the Data Commons graph.
+
+## Background
+
+Importing data into Data Commons boils down to taking source data and converting it into a format that can be consumed into the Data Commons graph. This section presents an example that does not require statistical data.
+
+Given tabular data like the following:
+
+|country_id  |  country_name	         |  continent_id|
+|-------|--------|---------|
+|USA	     |  United States of America |  northamerica|
+|IND	     |  India                    |	        asia|
+
+
+You can represent this data as a graph via subject-predicate-object "triples" that describe the node and edge relationships.
+```
+USA -- typeOf ------------> Country
+USA -- name --------------> United States of America
+USA -- containedInPlace --> northamerica
+```
+
+While you can express these triples via standards like [JSON-LD](https://json-ld.org/) or [RDFa](https://en.wikipedia.org/wiki/RDFa), the Data Commons format of choice is [MCF](https://github.com/datacommonsorg/data/tree/master/docs/mcf_format.md), which is concise and human readable. Here's the translated version of the example tabular data:
+
+```
+Node: USA
+typeOf: schema:Country
+name: "United States of America"
+containedInPlace: dcid:northamerica
+
+Node: IND
+typeOf: schema:Country
+name: "India"
+containedInPlace: dcid:asia
+```
+
+The `schema:` prefix represents the schema.org namespace, and `dcid:` represents the Data Commons namespace. For more information, see [scoping and namespaces in MCF](https://github.com/datacommonsorg/data/tree/master/docs/mcf_format.md#scoping-and-namespaces).
+
+The following sections explain how to convert [statistical data into graph representations](https://github.com/datacommonsorg/data/tree/master/docs/representing_statistics.md), as well as how to leverage templating to quickly map tabular data into [instance MCF](https://github.com/datacommonsorg/data/tree/master/docs/mcf_format.md#instance-mcf).
+
+
+## Importing The COVID Tracking Project's historical state data
```

Comment:
Maybe clarify in the title that this is an example import?

Reply:
I added 'Example' to the header.